static/js/search/place_results.tsx

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Result section for matching places
+ */
+
+import _ from "lodash";
+import React, { useEffect, useState } from "react";
+
+import { getPlaceNames } from "../utils/place_utils";
+
+const PLACE_REDIRECT_PREFIX = "/place/";
+const MAP_REDIRECT_PREFIX = "/tools/map#pd=";
+const SCATTER_REDIRECT_PREFIX = "/tools/scatter#epd=";
+const TIMELINE_REDIRECT_PREFIX = "/tools/timeline#place=";
+
+interface PlaceResultsProps {
+  places: string[];
+  selectedPlace: string;
+}
+
+export function PlaceResults(props: PlaceResultsProps): JSX.Element {
+  const [placeNames, setPlaceNames] = useState(null);
+
+  useEffect(() => {
+    const placeDcids = props.selectedPlace
+      ? [props.selectedPlace]
+      : props.places;
+    getPlaceNames(placeDcids)
+      .then((placeNames) => setPlaceNames(placeNames))
+      .catch(() => setPlaceNames({}));
+  }, [props]);
+
+  if (_.isEmpty(placeNames)) {
+    return <></>;
+  }
+
+  return (
+    <div className="search-results-place search-results-section">
+      {props.selectedPlace ? (
+        <div className="search-results-place-selected">
```

Comment:
Nit: we should match this behavior for stat var results too (when a user selects a particular stat var, we can display it highlighted on enter). You could add a TODO for now.

Reply:
done